.travis.yml

```diff
@@ -23,17 +23,17 @@ env:
     - PYTHON_VERSION=2.7
     - COVERALLS=false
     - NOSE_FLAGS="--processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50"
-    - NOSE_TEST_LIST1="analysis auxiliary coordinates core formats topology utils"
-    - NOSE_TEST_LIST2="lib"
-    - NOSE_COVERAGE1="coverage1"
-    - NOSE_COVERAGE2="coverage2"
-    - MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests ${NOSE_TEST_LIST1} ${NOSE_FLAGS}; python ./testsuite/MDAnalysisTests/mda_nosetests ${NOSE_TEST_LIST2} ${NOSE_FLAGS}"
+    - NOSE_TEST_LIST="analysis auxiliary coordinates core formats topology utils"
+    - PYTEST_LIST="lib"
+    - NOSE_COVERAGE="nose_coverage"
+    - PYTEST_COVERAGE="pytest_coverage"
+    - MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests ${NOSE_TEST_LIST} ${NOSE_FLAGS}; pytest ${PYTEST_LIST}"
     - SETUP_CMD=""
     - BUILD_CMD="pip install -v package/ && pip install testsuite/"
     - CONDA_DEPENDENCIES="mmtf-python nose=1.3.7 mock six biopython networkx cython joblib nose-timer"
     - CONDA_ALL_DEPENDENCIES="mmtf-python nose=1.3.7 mock six biopython networkx cython joblib nose-timer matplotlib netcdf4 scikit-learn scipy seaborn coveralls clustalw=2.1"
     # Install griddataformats from PIP so that scipy is only installed in the full build (#1147)
-    - PIP_DEPENDENCIES='griddataformats'
+    - PIP_DEPENDENCIES='griddataformats pytest pytest-cov'
```

Comment:
please add them do conda. They are both available on the conda-forge channel. This might also be messing up your pytest installation. But I'm not sure about that.